pkg/knuu/knuu_test.go

```diff
@@ -66,70 +66,84 @@ func TestNew(t *testing.T) {
 	}{
 		{
 			name:        "Default initialization",
-			options:     Options{},
+			options:     Options{TestScope: "test"},
```

Comment:
per my original comment, this case can now be reverted as `TestScope` shouldn't be required.